libs/sdk-ui-dashboard/src/presentation/alerting/DefaultAlertingManagementDialog/DefaultAlertingManagementDialog.tsx

```diff
@@ -81,16 +81,19 @@ export const AlertingManagementDialog: React.FC<IAlertingManagementDialogProps>
         ? defineMessage({ id: "dialogs.alerting.footer.title.short" }).id
         : defineMessage({ id: "dialogs.alerting.footer.title" }).id;
 
+    const titleElementId = React.useId();
```

Comment:
we need to be compatible with older React version, plse use our internal utility useId()